consensus/ising/proposerservice.go

```diff
@@ -717,6 +717,10 @@ func (ps *ProposerService) HandleBlockFloodingMsg(bfMsg *BlockFlooding, sender u
 			" hash: %s", votingHeight, height, BytesToHexString(blockHash.ToArrayReverse()))
 		return
 	}
+	if err := ledger.TimestampCheck(block.Header.Timestamp); err != nil {
```

Comment:
Seems that this check is not necessary since there is one in AddToCache?

Reply:
You are right.